index.html

```diff
@@ -35,5 +36,124 @@ <h2 class="profession">I’am a Software Developer</h2>
+                <ul class="languages">
+                    <li><a href="#">html</a>
+                    <li><a href="#">bootstrap</a>
+                    <li><a href="#">Ruby</a>
+                </ul>
+                <a class="access_button" href="#">See Project</a>
+            </div>
+            <div class="second_card">
+                <h3>Profesional Art Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="languages">
+                    <li><a href="#">html</a>
+                    <li><a href="#">bootstrap</a>
+                    <li><a href="#">Ruby</a>
+                </ul>
+                <a class="access_button" href="#">See Project</a>
+            </div>
+        </section>
+        <section class="containerTwo">
+            <div class="about">
+                <h2 class="title">About me</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need    coded, don’t hestiate to contact me.</p>
+                <a class="access_button" href="#">Get My Resume</a>
+                <ul class="details">
+                    <li><ul class="abilities">
+                        <li class="abilities_title">
+                            <img src="images/rectangle.svg" alt="Icon taken from figma">
+                            <h3>Language</h3>
+                        </li>
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>Html</li>
+                        <li>Css</li>
+                    </ul></li>
+                    <li><ul class="abilities">
+                        <li class="abilities_title">
+                            <img src="images/square.svg" alt="Icon taken from figma">
+                            <h3>Frameworks</h3>
+                        </li>
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul></li>
+                    <li><ul class="abilities">
+                        <li class="abilities_title">
+                            <img src="images/circle.svg" alt="Icon taken from figma">
+                            <h3>Skills</h3>
+                        </li>
+                        <li>Codekit</li>
+                        <li>Github</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul></li>
+                </ul>
+            </div>
+            <img class="image_background" src="images/imgback.png" alt="Image background taken from figma">
```

Comment:
[REQUIRED] Great work on adding a background image :+1:  However, I have to ask you to use `CSS` property `background-image` in this case please read further to discover why:

- [ ] As it is a background image,  adding it using `CSS` property `background-image` would make much sense since HTML `img` tag is used when the image has a value or is related to the content. In our case, it is just a background image and has no value to the content. Please refer to the below sample code:
```CSS
selector {
  background: #1c1a19 url(images/imgback.png) 90% 20% no-repeat;
  }
```
_note: you are more than welcome to use the above code, but please note that it is just a sample (example), not a solution, you need to adjust it if required_
- [ ] Please make sure you choose the correct `selector`, in this case, it should `.containerTwo`

We just want to make sure that you understand when to use one over the other.

Reply:
Thank you @Hope1226 I will make sure to apply those changes

Reply:
@Hope1226 I have a doubt. The README.md file is to be updated from this branch or the main one?. Is just that I did it from the main branch but I'm not sure if I have to add anything else